ansible/roles/riakcs/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- include: install.yml
+  when: riak_install == True
+
+- name: Set the Riak soft ulimit
+  lineinfile:
+    dest: /etc/security/limits.conf
+    state: present
+    line: '* soft nofile 4096'
+  tags: ulimit
+
+- name: Set the Riak hard ulimit
+  lineinfile:
+    dest: /etc/security/limits.conf
+    state: present
+    line: '* hard nofile 65536'
```

Comment:
made this `*` which seemed reasonable. this way it'll apply to the `riak` `riakcs` `stanchion` and `root` users